CMake fails with TJ_LIB: TJ_LIB-NOTFOUND

I tried running the install commands on a fresh Raspberry Pi OS Lite install (2020-12-02 release), but I wasn't able to make it past the cmake step. I see this failure:

```bash
-- Detecting C compile features - done
TJ_LIB: TJ_LIB-NOTFOUND
...
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
TJ_LIB
    linked by target "kvm_jpeg" in directory /home/pi/kvm/kvm_jpeg
```

I also see the following error in CMakeError.log:

```bash
/usr/bin/ld: CMakeFiles/cmTC_b2915.dir/CheckSymbolExists.cxx.o: in function `main':
CheckSymbolExists.cxx:(.text+0x48): undefined reference to `pthread_create'
collect2: error: ld returned 1 exit status
```

Full logs: https://gist.github.com/mtlynch/bd596f10dbf32acd05ad028cab725cf7

--- a/README.md
+++ b/README.md
@@ -105,7 +105,7 @@ Build and install the KVM software in the SSH session:
 
 ```shell
 sudo apt update
-sudo apt install -y git janus janus-dev cmake g++ libglib2.0-dev
+sudo apt install -y git janus janus-dev cmake g++ libglib2.0-dev libturbojpeg0-dev
 sudo systemctl disable janus
 
 cd ~

